src/main/java/picard/analysis/directed/CollectTargetedMetrics.java

```diff
@@ -165,6 +165,7 @@ protected int doWork() {
         if (THEORETICAL_SENSITIVITY_OUTPUT != null) {
             // Write out theoretical sensitivity results.
             final MetricsFile<TheoreticalSensitivityMetrics, ?> theoreticalSensitivityMetrics = getMetricsFile();
+            log.info("Calculating theoretical sentitivity at " + ALLELE_FRACTION.size() + " allele fractions. ");
```

Comment:
Do you want the space after "allele fractions."?
Also, if you are going to put this log here, can you put another one in CollectWgsMetrics?